# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build util-macros-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.63; am-1.11)
aclocal; autoconf; automake -a -c
./configure --prefix=/x11

# install
install -d $DESTDIR/x11/include/X11/bitmaps
install -m 644 1x1 $DESTDIR/x11/include/X11/bitmaps
install -m 644 2x2 $DESTDIR/x11/include/X11/bitmaps
install -m 644 black $DESTDIR/x11/include/X11/bitmaps
install -m 644 black6 $DESTDIR/x11/include/X11/bitmaps
install -m 644 box6 $DESTDIR/x11/include/X11/bitmaps
install -m 644 boxes $DESTDIR/x11/include/X11/bitmaps
install -m 644 calculator $DESTDIR/x11/include/X11/bitmaps
install -m 644 cntr_ptr $DESTDIR/x11/include/X11/bitmaps
install -m 644 cntr_ptrmsk $DESTDIR/x11/include/X11/bitmaps
install -m 644 cross_weave $DESTDIR/x11/include/X11/bitmaps
install -m 644 dimple1 $DESTDIR/x11/include/X11/bitmaps
install -m 644 dimple3 $DESTDIR/x11/include/X11/bitmaps
install -m 644 dot $DESTDIR/x11/include/X11/bitmaps
install -m 644 dropbar7 $DESTDIR/x11/include/X11/bitmaps
install -m 644 dropbar8 $DESTDIR/x11/include/X11/bitmaps
install -m 644 escherknot $DESTDIR/x11/include/X11/bitmaps
install -m 644 flagdown $DESTDIR/x11/include/X11/bitmaps
install -m 644 flagup $DESTDIR/x11/include/X11/bitmaps
install -m 644 flipped_gray $DESTDIR/x11/include/X11/bitmaps
install -m 644 gray $DESTDIR/x11/include/X11/bitmaps
install -m 644 gray1 $DESTDIR/x11/include/X11/bitmaps
install -m 644 gray3 $DESTDIR/x11/include/X11/bitmaps
install -m 644 grid16 $DESTDIR/x11/include/X11/bitmaps
install -m 644 grid2 $DESTDIR/x11/include/X11/bitmaps
install -m 644 grid4 $DESTDIR/x11/include/X11/bitmaps
install -m 644 grid8 $DESTDIR/x11/include/X11/bitmaps
install -m 644 hlines2 $DESTDIR/x11/include/X11/bitmaps
install -m 644 hlines3 $DESTDIR/x11/include/X11/bitmaps
install -m 644 icon $DESTDIR/x11/include/X11/bitmaps
install -m 644 keyboard16 $DESTDIR/x11/include/X11/bitmaps
install -m 644 left_ptr $DESTDIR/x11/include/X11/bitmaps
install -m 644 left_ptrmsk $DESTDIR/x11/include/X11/bitmaps
install -m 644 letters $DESTDIR/x11/include/X11/bitmaps
install -m 644 light_gray $DESTDIR/x11/include/X11/bitmaps
install -m 644 mailempty $DESTDIR/x11/include/X11/bitmaps
install -m 644 mailemptymsk $DESTDIR/x11/include/X11/bitmaps
install -m 644 mailfull $DESTDIR/x11/include/X11/bitmaps
install -m 644 mailfullmsk $DESTDIR/x11/include/X11/bitmaps
install -m 644 mensetmanus $DESTDIR/x11/include/X11/bitmaps
install -m 644 menu10 $DESTDIR/x11/include/X11/bitmaps
install -m 644 menu12 $DESTDIR/x11/include/X11/bitmaps
install -m 644 menu16 $DESTDIR/x11/include/X11/bitmaps
install -m 644 menu6 $DESTDIR/x11/include/X11/bitmaps
install -m 644 menu8 $DESTDIR/x11/include/X11/bitmaps
install -m 644 noletters $DESTDIR/x11/include/X11/bitmaps
install -m 644 opendot $DESTDIR/x11/include/X11/bitmaps
install -m 644 opendotMask $DESTDIR/x11/include/X11/bitmaps
install -m 644 plaid $DESTDIR/x11/include/X11/bitmaps
install -m 644 right_ptr $DESTDIR/x11/include/X11/bitmaps
install -m 644 right_ptrmsk $DESTDIR/x11/include/X11/bitmaps
install -m 644 root_weave $DESTDIR/x11/include/X11/bitmaps
install -m 644 scales $DESTDIR/x11/include/X11/bitmaps
install -m 644 sipb $DESTDIR/x11/include/X11/bitmaps
install -m 644 star $DESTDIR/x11/include/X11/bitmaps
install -m 644 starMask $DESTDIR/x11/include/X11/bitmaps
install -m 644 stipple $DESTDIR/x11/include/X11/bitmaps
install -m 644 target $DESTDIR/x11/include/X11/bitmaps
install -m 644 terminal $DESTDIR/x11/include/X11/bitmaps
install -m 644 tie_fighter $DESTDIR/x11/include/X11/bitmaps
install -m 644 vlines2 $DESTDIR/x11/include/X11/bitmaps
install -m 644 vlines3 $DESTDIR/x11/include/X11/bitmaps
install -m 644 weird_size $DESTDIR/x11/include/X11/bitmaps
install -m 644 wide_weave $DESTDIR/x11/include/X11/bitmaps
install -m 644 wingdogs $DESTDIR/x11/include/X11/bitmaps
install -m 644 woman $DESTDIR/x11/include/X11/bitmaps
install -m 644 xfd_icon $DESTDIR/x11/include/X11/bitmaps
install -m 644 xlogo11 $DESTDIR/x11/include/X11/bitmaps
install -m 644 xlogo16 $DESTDIR/x11/include/X11/bitmaps
install -m 644 xlogo32 $DESTDIR/x11/include/X11/bitmaps
install -m 644 xlogo64 $DESTDIR/x11/include/X11/bitmaps
install -m 644 xsnow $DESTDIR/x11/include/X11/bitmaps
install -d $DESTDIR/x11/lib/pkgconfig
install -m 644 xbitmaps.pc $DESTDIR/x11/lib/pkgconfig

# build.pc
install -d $DESTDIR/x11/share/pkgconfig
cat > $DESTDIR/x11/share/pkgconfig/xbitmaps-build.pc << EOF
Name: xbitmaps
Version: 1.1.0
Description: X bitmaps
Requires: $REQUIRES

devel=\\
/x11/include/X11/bitmaps/1x1 \\
/x11/include/X11/bitmaps/2x2 \\
/x11/include/X11/bitmaps/black \\
/x11/include/X11/bitmaps/black6 \\
/x11/include/X11/bitmaps/box6 \\
/x11/include/X11/bitmaps/boxes \\
/x11/include/X11/bitmaps/calculator \\
/x11/include/X11/bitmaps/cntr_ptr \\
/x11/include/X11/bitmaps/cntr_ptrmsk \\
/x11/include/X11/bitmaps/cross_weave \\
/x11/include/X11/bitmaps/dimple1 \\
/x11/include/X11/bitmaps/dimple3 \\
/x11/include/X11/bitmaps/dot \\
/x11/include/X11/bitmaps/dropbar7 \\
/x11/include/X11/bitmaps/dropbar8 \\
/x11/include/X11/bitmaps/escherknot \\
/x11/include/X11/bitmaps/flagdown \\
/x11/include/X11/bitmaps/flagup \\
/x11/include/X11/bitmaps/flipped_gray \\
/x11/include/X11/bitmaps/gray \\
/x11/include/X11/bitmaps/gray1 \\
/x11/include/X11/bitmaps/gray3 \\
/x11/include/X11/bitmaps/grid16 \\
/x11/include/X11/bitmaps/grid2 \\
/x11/include/X11/bitmaps/grid4 \\
/x11/include/X11/bitmaps/grid8 \\
/x11/include/X11/bitmaps/hlines2 \\
/x11/include/X11/bitmaps/hlines3 \\
/x11/include/X11/bitmaps/icon \\
/x11/include/X11/bitmaps/keyboard16 \\
/x11/include/X11/bitmaps/left_ptr \\
/x11/include/X11/bitmaps/left_ptrmsk \\
/x11/include/X11/bitmaps/letters \\
/x11/include/X11/bitmaps/light_gray \\
/x11/include/X11/bitmaps/mailempty \\
/x11/include/X11/bitmaps/mailemptymsk \\
/x11/include/X11/bitmaps/mailfull \\
/x11/include/X11/bitmaps/mailfullmsk \\
/x11/include/X11/bitmaps/mensetmanus \\
/x11/include/X11/bitmaps/menu10 \\
/x11/include/X11/bitmaps/menu12 \\
/x11/include/X11/bitmaps/menu16 \\
/x11/include/X11/bitmaps/menu6 \\
/x11/include/X11/bitmaps/menu8 \\
/x11/include/X11/bitmaps/noletters \\
/x11/include/X11/bitmaps/opendot \\
/x11/include/X11/bitmaps/opendotMask \\
/x11/include/X11/bitmaps/plaid \\
/x11/include/X11/bitmaps/right_ptr \\
/x11/include/X11/bitmaps/right_ptrmsk \\
/x11/include/X11/bitmaps/root_weave \\
/x11/include/X11/bitmaps/scales \\
/x11/include/X11/bitmaps/sipb \\
/x11/include/X11/bitmaps/star \\
/x11/include/X11/bitmaps/starMask \\
/x11/include/X11/bitmaps/stipple \\
/x11/include/X11/bitmaps/target \\
/x11/include/X11/bitmaps/terminal \\
/x11/include/X11/bitmaps/tie_fighter \\
/x11/include/X11/bitmaps/vlines2 \\
/x11/include/X11/bitmaps/vlines3 \\
/x11/include/X11/bitmaps/weird_size \\
/x11/include/X11/bitmaps/wide_weave \\
/x11/include/X11/bitmaps/wingdogs \\
/x11/include/X11/bitmaps/woman \\
/x11/include/X11/bitmaps/xfd_icon \\
/x11/include/X11/bitmaps/xlogo11 \\
/x11/include/X11/bitmaps/xlogo16 \\
/x11/include/X11/bitmaps/xlogo32 \\
/x11/include/X11/bitmaps/xlogo64 \\
/x11/include/X11/bitmaps/xsnow \\
/x11/lib/pkgconfig/xbitmaps.pc
EOF

exit

# patch
--- xbitmaps-1.1.0.orig/configure.ac
+++ xbitmaps-1.1.0/configure.ac
@@ -3,8 +3,6 @@
 AM_INIT_AUTOMAKE([dist-bzip2])
 AM_MAINTAINER_MODE
 
-AM_CONFIG_HEADER(config.h)
-
 # Require xorg-macros 1.3 or later: XORG_DEFAULT_OPTIONS
 m4_ifndef([XORG_MACROS_VERSION],
 	  [m4_fatal([must install xorg-macros 1.3 or later before running autoconf/autogen])])

# source
https://xorg.freedesktop.org/releases/individual/data/xbitmaps-1.1.0.tar.gz
